src/components/InputTodo.js

```diff
@@ -1,37 +1,44 @@
-import React, { useState } from "react"
-import { FaPlusCircle } from "react-icons/fa";
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+import PropTypes from 'prop-types';
 
 const InputTodo = (props) => {
-    const [title, setTitle] = useState("");
+  const [title, setTitle] = useState('');
 
-    const onChange = e => {
-        setTitle(e.target.value);
-    };
+  const { addTodoProps } = props;
 
-    const handleSubmit = e => {
-        e.preventDefault();
-        if (title.trim()) {
-            props.addTodoProps(title)
-            setTitle("");
-        } else {
-            alert("Please write item")
```

Comment:
Please consider implementing the form validation using a render since `window.alert` does not conform to the [Javascript Best Practises](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

Reply:
Thanks for the comment @JonahKayizzi but this was addressed already, I believe you didn't review the right version